src/Sylius/Bundle/CoreBundle/Announcer/BatchedVariantsUpdateAnnouncer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Announcer;
+
+use Sylius\Component\Product\Command\UpdateBatchedVariants;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+final class BatchedVariantsUpdateAnnouncer implements BatchedVariantsUpdateAnnouncerInterface
```

Comment:
```suggestion
final class BatchOfVariantsUpdateAnnouncer implements BatchOfVariantsUpdateAnnouncerInterface
```
or
```suggestion
final class VariantsBatchUpdateAnnouncer implements VariantsBatchUpdateAnnouncerInterface
```

Reply:
However, after looking at the content of this class, I wonder if it should be `*Announcer`, because, by that name, I expect an event rather than a command. It is more of a processor/process manager (I have no better name right now) that takes over some data and triggers the next processes

Reply:
That was my idea, but you are absolutely correct - Announcer is wrong wording, as these are commands not events. What would you say for ProcessManager or Commander? 

```suggestion
final class BatchedVariantsUpdateProcessManager implements VariantsUpdateProcessManagerInterface
```

My proposal are as follows:
* Add `Commander` or `ProcessManager` suffix for this class and its interface
* Remove `Batched` prefix from the interface name - it is implementation detail and interface does not need to know about it
* Remove `Batched` prefix from the command name - we don't need to know if it was batched or not. It doesn't matter. What matters is, that it contains more then 1 variant, therefore `UpdateVariants` should be a good candidate